charts/internal/seed-controlplane/charts/csi-vsphere/values.yaml

```diff
@@ -14,7 +16,8 @@ password: mypassword
 port: 443
 datacenters: dc1
 insecureFlag: true
-resizerEnabled: true
+resizerEnabled: false
+csi2: false
 loggerLevel: PRODUCTION
 resources:
```

Comment:
I'm not sure if these resource requests and limits are good enough for controller and syncer containers in a scaled environment. We are working on coming up with recommendations on this.